internal/jobexecutor/job_executor_test.go

```diff
@@ -736,11 +736,10 @@ func TestJobExecutor_Execute(t *testing.T) {
 }
 
 type testMiddleware struct {
+	rivertype.MiddlewareDefaults
 	work func(ctx context.Context, job *rivertype.JobRow, next func(context.Context) error) error
 }
 
-func (m *testMiddleware) IsMiddleware() bool { return true }
```

Comment:
This could also be considered a minor benefit. Previously, adding a middleware to tests for an internal package like `jobexecutor` here forced you to provide your implementation because `MiddlewareDefaults` was in `river`, and you can't import that lest you introduce a cycle. Putting `MiddlewareDefaults` in `rivertype` means it's all good.